package.json

```diff
@@ -52,25 +52,25 @@
     "TypeScript"
   ],
   "dependencies": {
-    "eslint-config-standard": "^12.0.0"
+    "@typescript-eslint/parser": "^1.13.0",
+    "eslint-config-standard": "^13.0.1"
   },
   "peerDependencies": {
-    "eslint": "^5.8.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^9.0.0",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "@typescript-eslint/eslint-plugin": "^1.4.2",
-    "@typescript-eslint/parser": "^1.4.2"
+    "eslint": ">=6.0.1",
+    "eslint-config-standard": ">=13.0.1",
+    "eslint-plugin-import": ">=2.18.0",
+    "eslint-plugin-node": ">=9.1.0",
+    "eslint-plugin-promise": ">=4.2.1",
+    "eslint-plugin-standard": ">=4.0.0",
+    "@typescript-eslint/eslint-plugin": ">=1.10.1"
   },
   "devDependencies": {
     "@commitlint/cli": "^8.1.0",
     "@commitlint/config-conventional": "^8.1.0",
     "@commitlint/travis-cli": "^8.1.0",
     "@types/node": "^12.6.9",
-    "@typescript-eslint/eslint-plugin": "^1.9.0",
-    "@typescript-eslint/parser": "^1.11.0",
-    "eslint": "^5.16.0",
+    "@typescript-eslint/eslint-plugin": "^1.10.2",
```

Comment:
I did not intend to update the TypeScript plugin. Yet, the current version seems to crash with some error when using the latest parser.